configs/workflows-mt.yaml

```diff
@@ -0,0 +1,377 @@
+      output_read_bases: "{outputs.stats.output_read_bases}"
+      output_read_count: "{outputs.stats.output_read_count}"
+      type: nmdc:ReadQcAnalysisActivity
+    Outputs:
+      - output: filtered_final
+        name: Reads QC result fastq (clean data)
+        data_object_type: Filtered Sequencing Reads
+        description: "Reads QC for {id}"
+      - output: filtered_stats_final
+        name: Reads QC summary statistics
+        data_object_type: QC Statistics
+        description: "Reads QC summary for {id}"
+      - output: rqc_info
+        name: File containing read filtering information
+        data_object_type: Read Filtering Info File
+        description: "Read filtering info for {id}"
+      - output: filtered_ribo_final
+        name: Fastq file containing filtered ribosomal sequences
+        data_object_type: rRNA Filtered Sequencing Reads
+        description: "rRNA fastq for {id}"
+
+  - Name: Reads QC Interleave
+    Type: nmdc:ReadQcAnalysisActivity
+    Enabled: True
+    Git_repo: https://github.com/microbiomedata/metaT_ReadsQC
+    Version: v0.0.3
+    Collection: read_qc_analysis_activity_set
+    WDL: interleave_rqcfilter.wdl
+    Input_prefix: nmdc_rqcfilter
+    Inputs:
+      proj: "{activity_id}"
+      input_fastq1: do:Metagenome Raw Read 1
+      input_fastq2: do:Metagenome Raw Read 2
+    Filter Input Objects:
+    - Metagenome Raw Read 1
+    - Metagenome Raw Read 2
+    Predecessors:
+    - Sequencing Noninterleaved
+    Activity:
+      name: "Read QC Activity for {id}"
+      input_read_bases: "{outputs.stats.input_read_bases}"
+      input_read_count: "{outputs.stats.input_read_count}"
+      output_read_bases: "{outputs.stats.output_read_bases}"
+      output_read_count: "{outputs.stats.output_read_count}"
+      type: nmdc:ReadQcAnalysisActivity
+    Outputs:
+      - output: filtered_final
+        name: Reads QC result fastq (clean data)
+        data_object_type: Filtered Sequencing Reads
+        description: "Reads QC for {id}"
+      - output: filtered_stats_final
+        name: Reads QC summary statistics
+        data_object_type: QC Statistics
+        description: "Reads QC summary for {id}"
+      - output: rqc_info
+        name: File containing read filtering information
+        data_object_type: Read Filtering Info File
+        description: "Read filtering info for {id}"
+      - output: rrna_fastq_final #placeholder until https://github.com/microbiomedata/metaT_ReadsQC/issues/5 is resolved
+        name: Fastq file containing filtered ribosomal sequences             
+        data_object_type: rRNA Filtered Sequencing Reads
+        description: "rRNA fastq for {id}"
+ 
+  - Name: Metatranscriptome Assembly
+
+    Type: nmdc:MetatranscriptomeAssembly
+    Enabled: True
+    Git_repo: https://github.com/microbiomedata/metaT_Assembly
+    Version: v0.0.1
+    WDL: metaT_assembly.wdl
+    Collection: metagenome_assembly_set
+    Predecessors:
```

Comment:
Since metaG and metaT both have the same predecessor here, we will probably need to add some filter on the omics type that these are connected to.  Perhaps we could add a new field to all workflows that is the omics processing type.

Reply:
Updated so workflow names are unique and added Analyte Category to  which is in line with berkeley refactor nomeclature to workflows.yaml and workflows-mt.yaml